### PR TITLE
Add features for different ecs spans

### PR DESCRIPTION
**Pull request by hymm**
# Objective

- It's often useful when benchmarking changes to schedules and the executor to disable the system level spans.

## Solution

- When running examples you can now enable specific spans in bevy_ecs. The following command would enable all the spans
```text
cargo run --example many_foxes -F bevy_internal/trace_tracy,bevy_internal/app_spans,bevy_internal/schedule_spans,bevy_internal/system_task_spans,bevy_internal/system_spans,bevy_internal/par_iter_spans
```
- I didn't expose top level features for this. It's probably useful for users, but I'm not sure about adding 10 different features for all the different spans. 
- Also the system task spans are no longer enabled by default. For general users these mostly just duplicated information.
---

## Changelog

- added features for bevy spans for internal benchmarking

### CODE REVIEW

File: crates/bevy_ecs/Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,11 @@ keywords = ["ecs", "game", "bevy"]
 categories = ["game-engines", "data-structures"]
 
 [features]
-trace = []
+trace = [ "schedule_spans", "system_spans", "par_iter_spans"]
```
Comment (BD103) [inline crates/bevy_ecs/Cargo.toml:13]:
```suggestion
trace = ["schedule_spans", "system_spans", "par_iter_spans"]
```

**Review by BD103 — COMMENTED** [review]
This is interesting. Would it be useful to have a `bevy_internal/all_spans` feature that enables all of the spans? It would shorten the command and somewhat keep prior behavior for general debugging.

Additionally, should this be documented in `CONTRIBUTING.md` or something similar? I feel like it would be useful for others to know about.

**Comment by mockersf** [comment]
it should already be possible to selectively enable/disable spans, by changing the log level of the module. Most of the spans are using `info_span` so are at info level, set their module to warn and you shouldn't have the spans

**Comment by mockersf** [comment]
you can configure logging level by module, not just by crate, so I guess it would be `bevy_ecs::schedule::schedule`

**Comment by hymm** [comment]
ah nice. `RUST_LOG=bevy_ecs::schedule::schedule=info,bevy_app=info,bevy_render=info` seems to get me what I want when I'm testing scheduling changes. I'll try to find a place to document it and some of the other spans.

**Pull request closed without merging**